src/utils/Guild.js

```diff
@@ -18,9 +23,7 @@ function parseHistory(historyData) {
           .map((x) => parseInt(x, 10))
       ) || undefined,
     exp: x[1] || 0,
-    totalExp: Object.values(historyData)
```

Comment:
`Object.values(historyData).reverse()`

Reply:
Get I get an explanation of what you're requesting here? This code was removed

Reply:
You should just be able to use the original code and just reverse it first. There shouldnt be a need for a new function

Reply:
I mean sure, but this just makes it horrendously unreadable lol